app/src/androidTest/java/com/google/samples/apps/sunflower/GardenActivityTest.kt

```diff
@@ -52,13 +52,14 @@ class GardenActivityTest {
     @Test fun onRotate_NavigationStaysOpen() {
         clickOnHomeIconToOpenNavigationDrawer()
 
-        // Rotate device to landscape
-        activityTestRule.activity.requestedOrientation = SCREEN_ORIENTATION_LANDSCAPE
-        checkDrawerIsOpen()
+        // Rotate device to different orientations
+        with(UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())) {
+                setOrientationLeft()
+                checkDrawerIsOpen()
 
-        // Rotate device back to portrait
-        activityTestRule.activity.requestedOrientation = SCREEN_ORIENTATION_PORTRAIT
-        checkDrawerIsOpen()
+                setOrientationRight()
```

Comment:
Add comment: Rotate device back to portrait

Reply:
👍